src/styles/null.scss

```diff
@@ -0,0 +1,9 @@
+html {
```

Comment:
Strange naming ;)
Let's change it to reset.scss

Reply:
done